solcjs

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/env node
+
+var fs = require('fs');
+var solc = require('./index.js');
+// FIXME: remove annoying exception catcher of Emscripten
+//        see https://github.com/chriseth/browser-solidity/issues/167
+process.removeAllListeners('uncaughtException');
```

Comment:
Better to keep the shebang in a separate file. Then index.js can still be required when used programmatically.